ci/scripts/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash -eux
 
 pushd dp-adot-collector
+  make build
```

Comment:
This isn't needed as there isn't a makefile to run the build command in this project